test/recipes/80-test_ssl.t

```diff
@@ -626,22 +644,39 @@ sub testssl {
 	  is(run(test([@ssltest, "-bio_pair", "-tls1", "-alpn_client", "baz", "-alpn_server", "bar,foo"])), 0,
              "Testing ALPN with protocol mismatch, expecting failure");
 
-	SKIP: {
-	    skip "skipping SRP tests", 4
-		if $no_srp;
+	  # ALPN + SNI
+	  ok(run(test([@ssltest, "-bio_pair", "-tls1",
+		       "-alpn_client", "foo,bar", "-sn_client", "foo",
```

Comment:
Could you use different foobars for ALPN and SN, so that it becomes easier to read which selection we're testing?